test/test_operators.py

```diff
@@ -133,5 +134,30 @@ def avg_pool2d_strides_ceil_mode_pad(a):
         assert torch.allclose(ref_out, tvm_out, rtol=0.01, atol=0.01)
 
 
+    # Known bug -- ceil_mode=True sometimes has mismatched shapes
+    @TVMTest.given(
+        shape=TVMTest.rand_shape(rank=4, min_dim=4),
+        stride=TVMTest.rand_list(TVMTest.rand_int(1, 2), 2),
+    )
+    def test_max_pool2d(self, shape, stride):
+        X = torch.rand(shape)
+        def max_pool2d(a):
+            return F.max_pool2d(a, 3) + 2.0
+
+        def max_pool2d_strides_padding_ceil_mode(a):
+            return F.max_pool2d(
+                a, 2, stride=stride, padding=1, ceil_mode=False
+            )
+
+        # TODO: fix the unstableness when ceil_mode=True case
```

Comment:
To support ceil_mode you can adjust the padding value to pass to tvm.

Reply:
Oh in fact relay do have a `ceil_mode`, and their ways of handling padding seems to be same with what pytorch did (but it turned out different which needs some debugging), so i am not sure if adjust the padding value here will help.